### DISCUSSION TITLE
Welcome to zencart Discussions!

### DISCUSSION THREAD

**Discussion by zcwilt** [zencart/zencart#4526] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by scottcwilson** [comment]
My name is Scott and I take care of the Zen Cart plugins and documentation.

**Comment by g2ktcf** [comment]
My name is Chris.  I am learning.  I am a mechanical engineer and project manager for my day job.  I have done programming in multiple languages and platforms over the years.  I am still learning the ZC "format" and process so please bear with me when I ask a thousand questions!  I am still setting up a dev environment to do some testing on #2934 .  I have been running one or two ZC stores since 2011.

**Comment by scottcwilson** [comment]
@barco57 @mc12345678 You have outstanding invitations to the Zen Cart organization; please accept them.

**Comment by torvista** [comment]
Hi,
I’m Steve, a business owner (not a billing developer) but with a long background in tech, starting with PETs with cassette loaders.

I started using Zen Cart in 2011, a dual-language English-Spanish store and so have become the maintainer of the Spanish language pack for my troubles, along with various other add-ons in GitHub.

I have spent a totally unjustifiable amount of time improving most of the mods/ideas I’ve used from the Plugins and trying to correct errors detected along the way in line with my poor knowledge / php experience…hence I initially chose the icon of Trivial Pursuit as that was what I (and others I imagine) felt I was doing…pursuing the trivial /chasing the very-low hanging fruit instigated by that programmers friend, the eternally complaining HTML Validator.
Still, these days ZC is almost error free on that front, for which I’ll take some credit (in lieu of cash).

Thanks to the trivial, I’ve more confidence in the not-so trivial, and the grumpiness/impatience that comes with age to stick my ideas/mods into more aspects of the project, ostensibly for altruistic motives but really to make my upgrades easier!

I would like to see my investment in ZC vindicated by the project reclaiming the ground lost to the competition over the last ten years, which is a subject long overdue for discussion/action that may finally have a outlet in this space.

**Comment by Odusplits** [comment]
How do I make a working shopping cart

**Reply by dbltoe** [reply]
Download the latest file from https://github.com/zencart/zencart/archive/refs/tags/v1.5.7d.zip.  There is a docs folder included in the files.

Also, look at https://docs.zen-cart.com/user/first_steps/ for guidance.

Bookmark the [forum](https://www.zen-cart.com/forum.php) for help when you need it.

Checkout Services if you need hosting, programming, or payment options.

**Comment by dbltoe** [comment]
My name is John.

I've been around since before Al Gore invented the Internet.  Mostly self-taught, I started with a Sinclair ZX-80 clocked at 3.25MHz with 1 KB of static ram and 4 KB of ROM.  Hard drive?  Nope.  Everything went on a cassette tape.  The modem was 300 baud.

Stationed on an island at the end of the Aleutian Islands, I used it to keep from becoming an alcoholic, smoker, or both.  It kept me busy.  Sinclair Basic was the code.  Sinclair Basic was far superior to the Commodore's Basic but somehow Commodore won out.

I graduated thru 286, 386, and 486, all self-taught.  Most of my time was in HTML, then ASP, and finally ColdFusion.

Websites were my puzzles to solve and the more tools the more I loved it.

A foolish "I can do that" led me to be hired by TriCare to make their website 508 compliant.  That's the precursor to ADA and WCAG and was twenty years ago.  Their website was all ColdFusion in those days and I figured the added requirements would be a piece of cake. It took six months of hard work.  And, no, their website is not compliant today. 👎 

The wife and I have side businesses other than myZenCartHost.com and, in 2003, I was starting to design my own e-commerce website in ColdFusion.  There were a couple of sites using ColdFusion but they were commercial and I started looking at OsCommerce.  Not long after that, ajeh and others decided to fork over to Zen Cart.  I followed and have been happy ever since.

I'm still sketchy on PHP but CSS and Accessibility are my specialties.

I've maintained my presence on the Forum because I'm a teacher at heart.  I'm known to "hijack" threads to point out something that might break a site even if it had nothing to do with the original question.

I try to answer what I can so the developers can do their thing.

I believe that SEO, user-friendly links, and paid clicks are a waste of time and money.  A site with little or no code errors and proper product descriptions can make it to the first page in any SE.

I also believe that making a website comply with ADA/WCAG does not mean it has to be dull.

When I have time these days, I am working on trying to make ZC ADA/WCAG compliant out of the box.  So far, proposed changes will get ZC four 100s on Chrome's Lighthouse for desktop.  There is not another cart out there that can claim that.

There have been a lot of changes over the last 18 years.  I'm glad to contribute in some small way to make it better every day.
